lib/mongo/topology_description.ex

```diff
@@ -11,8 +11,8 @@ defmodule Mongo.TopologyDescription do
   alias Mongo.ReadPreference
 
   # see https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#topologydescription
-  @type type :: :unknown | :single | :replica_set_no_primary |
-                :replica_set_with_primary | :sharded
+  @type type :: :"unknown()" | :"single()" | :"replica_set_no_primary()" |
+                :"replica_set_with_primary()" | :"sharded()"
```

Comment:
Yes!.. but I got some errors... and now I don't... really don't know what was happening, but now it seems to work well.
Thanks!